pkg/agent/openflow/client.go

```diff
@@ -407,9 +408,12 @@ func (c *client) InstallEndpointFlows(protocol binding.Protocol, endpoints []pro
 	for _, endpoint := range endpoints {
 		var flows []binding.Flow
 		endpointPort, _ := endpoint.Port()
-		endpointIP := net.ParseIP(endpoint.IP()).To4()
+		endpointIP := net.ParseIP(endpoint.IP())
```

Comment:
Could we ensure the endpoints all have the same IP family? I think we can validate the IP family out of the loop.
Furthermore, if the endpoints all have the same IP family, can we pass the IP family as an argument explicitly instead of checking it?

Reply:
Sorry, I should point to the next line. I just feel we can avoid that 'heavy' checking.

Reply:
Done